web/src/store/GlobalStore.js

```diff
@@ -45,6 +49,9 @@ export const INITIAL_STATE = {
 function reducer(state, action) {
   switch (action.type) {
     case actions.UPDATE_STATE:
+      if (has(action.payload, "calculatedFilters.projects")) {
```

Comment:
Best to avoid side effects inside a reducer, we usually want it to handle state update and nothing else. I would put this in the provider that provides this reducer, or in the components you are dispatching from.